expression/column.go

```diff
@@ -240,6 +240,21 @@ func (col *Column) VecEvalInt(ctx sessionctx.Context, input *chunk.Chunk, result
 
 // VecEvalReal evaluates this expression in a vectorized manner.
 func (col *Column) VecEvalReal(ctx sessionctx.Context, input *chunk.Chunk, result *chunk.Column) error {
+	n := input.NumRows()
+	src := input.Column(col.Index)
+	if col.GetType().Tp == mysql.TypeFloat {
+		result.ResizeFloat64(n, false)
+		f32s := src.Float32s()
+		f64s := result.Float64s()
+		for i := range f32s {
+			if src.IsNull(i) {
+				result.SetNull(i, true)
```

Comment:
Could we copy the whole `nullBitmap` here? It may improve performance.

Reply:
We have to introduce some new methods on `Column` to do it like `CopyNulls(otherCol)`, which is a little ad-hoc?
What are your opinions? @zz-jason @XuHuaiyu

Reply:
How's the performance improved if we did it when all rows are null or not?

Reply:
I also think the performance needs to be benchmark.

Reply:
How about leaving it as a further optimization since:
1) the performance gain is decided by the data distribution but we don't know it now, 
2) the performance now is already faster than before.

And if we want to introduce more methods to manipulate null bitmaps, we should design it carefully in another PR.
For example, in addition to `CopyNulls`, do we need to introduce more methods like `OrNulls(otherCol, begin, end)`, `AndNulls(otherCol, begin, end)`?

Reply:
A TODO comment is added here and I will speed up the way to manipulate null-bitmaps in the next few PRs.